### ISSUE TITLE
Error types breadcrumbs

### ISSUE THREAD

**Issue by nasirDoe** [harlan-zw/nuxt-seo-kit#79]
### Describe the bug

when i enable typescript check in `nuxt.config`

```js
typescript: {
    typeCheck: true,
 }
```
<img width="1254" alt="Screenshot 2023-08-09 at 16 30 35" src="https://github.com/harlan-zw/nuxt-seo-kit/assets/16643567/a1b6ff51-63c5-4137-88c5-c6891135f65b">

### Reproduction

-

### System / Nuxt Info

```Shell
- nuxt 3.6.5
- OSX Ventura 13.5
- Brave (chrome)
```

**Comment by ennioVisco** [comment]
Duplicate of: https://github.com/harlan-zw/nuxt-seo-kit/issues/40

**Comment by harlan-zw** [comment]
Most of these should be fixed in the latest, can track in #40 anything missed
